lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -66,6 +66,29 @@ class EditingState {
         baseOffset = flutterEditingState['selectionBase'],
         extentOffset = flutterEditingState['selectionExtent'];
 
+  /// Creates an [EditingState] instance using values from the editing element
+  /// in the dom.
+  ///
+  /// This element can be a InputElement or a TextAreaElement depending on the
+  /// [InputType] of the text field.
+  factory EditingState.fromDomElement(html.HtmlElement domElement) {
```

Comment:
Why not call `configureDomElementFromEditingState`?

Reply:
`EditingState.fromDomElement` and `EditingState.configureDomElementFromEditingState` do opposite things. The former reads the state from the DOM element. The latter sets the state on the DOM element.

Reply:
ack.